pkg/init/init

```diff
@@ -1,9 +0,0 @@
-#!/bin/sh
-
-/bin/mount -t tmpfs tmpfs /mnt
-
-/bin/cp -a / /mnt 2>/dev/null
-
-/bin/mount -t proc -o noexec,nosuid,nodev proc /proc
```

Comment:
Yes, it is not needed, it is done as the first line of the init script in `/bin/rc.init`